Week-2/Homework/hw.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Classroom HW</title>
+    <style>
+    table {
+        font-family: arial, sans-serif;
+        border-collapse: collapse;
+        width: 100%;
+    }
+
+    td, th {
+        border: 1px solid #dddddd;
+        text-align: left;
+        padding: 8px;
+    }
+
+    tr:nth-child(even) {
+        background-color: #dddddd;
+    }
+    </style>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+  </head>
+  <body>
+    <h2>Input a name!</h2>
+    <input type="text" id="name"></input>
+  </br>
+    <button onclick="createStudent();">Go</button>
+    <table id = "myTable">
+      <tr>
+        <th>Student Name</th>
+        <th>Age</th>
+        <th>Grades</th>
+        <th>Average</th>
+      </tr>
+    </table>
+    <script>
+    let grade = [];
+    let y = 0;
+
+    let createStudent = () => {
+      let student = [];
+      for (let member in student) {
+        delete student[member];
```

Comment:
no idea what this is about. You define student to be an empty list, and then loop through it? It seems like this loop does nothing.